README.md

```diff
@@ -66,14 +103,54 @@ of data in specific buckets. Latency can be good example of a histogram metric.
 
 ## I
 
+### Incident
+
+An incident refers to an unexpected event that causes a system or service to fail or degrade. Examples of incidents can include server crashes, network outages, data center failures, or security breaches.
+
 ## J
 
 ## K
 
+### Kubernetes
+
+Kubernetes is a popular open-source platform for managing containerized workloads and services. Kubernetes provides features for deploying, scaling, and managing containerized applications.
+
+### KPIs
+
+KPIs (Key Performance Indicator) are specific metrics used to track progress towards a specific goal or objective. In the context of SRE, KPIs can be used to measure the performance and reliability of a system.
+
 ## L
 
+### Log
+
+Logs are an record of activities performed by a system or service over a set period of time. Logs help SRE teams to track behaviours of a system. 
```

Comment:
```suggestion
Logs record activities a system, service, or tool performs over a set time. They are easy to adopt but run into standardization challenges because different tools and languages can have different logging patterns. Logs can grow quickly but are extremely important for debugging a problem once identified.
```